src/States.jl

```diff
@@ -1,10 +1,13 @@
 # Contains the structs that are evolved by the river model, and methods to instantiate them
-export RiverState
+export RiverState, HillslopeChannelRiverState
 
-struct RiverState{AM <: AbstractMatrix}
-    hillslope::AM # instantaneous
-    channel::AM # instantaneous
-end
+abstract type RiverState end
 
-# initialize method
-# initialize_state(river_model::HCRM) where {HCRM <: HillslopChannelRiverModel} end
+struct HillslopeChannelRiverState <: RiverState
+    "Hillslope state history [Dict(id -> Matrix: time-within-lag (end=current) x basin)]"
```

Comment:
Shouldnt this be basin_id -> vector (of length days in time window)? Im confused by the "x basin" part

Reply:
Done! You are correct!